docker-compose.yml

```diff
@@ -1,10 +1,9 @@
-# docker-compose.yml
 version: '3'
 networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: todoapp:${VERSION:-2.0.0}
```

Comment:
Ensure that the `VERSION` environment variable is set and that the image `todoapp` with the specified version exists in your Docker registry. If not, the default version `2.0.0` will be used, which may not exist.